test/functional/address_types.py

```diff
@@ -104,10 +116,26 @@ def test_address(self, node, address, multisig, typ):
             # Unknown type
             assert(False)
 
+    def test_change_output_type(self, node_sender, destinations, expected_type):
+        txid = self.nodes[node_sender].sendmany("",  dict.fromkeys(destinations, 0.001))
```

Comment:
suggest you make these named arguments so it's clear what the `""` is for.

Reply:
Done